lib/zendesk_apps_support/validations/secure_settings.rb

```diff
@@ -43,6 +50,18 @@ def secure_settings_warning
             link: 'https://developer.zendesk.com/apps/docs/developer-guide/using_sdk#using-secure-settings'
           )
         end
+
+        def secure_param_without_scopes?(parameter)
+          !parameter.scopes&.any?
+        end
+
+        def no_scopes_warning(param_names)
+          I18n.t(
+            'txt.apps.admin.error.app_build.translation.secure_parameters_with_no_scopes_in_manifest',
+            params: param_names.join(', '),
```

Comment:
Consider using `'txt.apps.admin.error.app_build.listing_comma'` translation like in https://github.com/zendesk/zendesk_apps_support/blob/6d6aa8c475a51e1c75568020d7a6939d200b3738/lib/zendesk_apps_support/validations/manifest.rb#L153